backend/benefit/users/models.py

```diff
@@ -0,0 +1,8 @@
+import uuid
+
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
Hmn, I forgot to ask, should we add a prefix to the db table name?

Reply:
Good catch! Added "bf_" prefix so table name would be "bf_users_user". (I *think* this was the prefix mentioned in Slack chat, but I can't find the message - if we had agreed on a different prefix, then ping me?

Reply:
yes it is the bf_ prefix :)